app/presenters/two_factor_authentication/auth_app_selection_presenter.rb

```diff
@@ -8,5 +8,13 @@ def method
     def security_level
       I18n.t('two_factor_authentication.two_factor_choice_options.secure_label')
     end
+
+    def disabled?
+      !user.nil? && user.auth_app_configurations.any?
```

Comment:
I think Rubocop gets mad a `.nil?` checks, but we could also use the null-safe operator I think on this:

```suggestion
      user&.auth_app_configurations&.any?
```